lib/client/search_pager.js

```diff
@@ -29,19 +31,20 @@ const PagedControl = require('../controls/paged_results_control.js')
  *                will be emitted (and 'end' will not be).  By listening to
  *                'pageError', a successful search that lacks paging will be
  *                able to emit 'end'.
- * 3. search    - Emitted as an internal event to trigger another client search.
  */
 function SearchPager (opts) {
   assert.object(opts)
   assert.func(opts.callback)
   assert.number(opts.pageSize)
+  assert.func(opts.sendRequest)
```

Comment:
Does this alter the public API and introduce a breaking change?

Reply:
The search event was clearly marked as internal and only used in one place.